src/js/vue-plugins/router.js

```diff
@@ -83,6 +84,12 @@ const addDocumentTypeView = function (def, viewComponent, editionComponent) {
     component: DocumentsView,
   });
 
+  routes.push({
+    path: '/print/' + def.documentType + 's',
```

Comment:
I wonder if reversing the path wouldn't be more consistent?
```suggestion
    path: def.documentType + 's/print',
```